analysed-packages/logrotate/version-3.20.1/logrotate-3.20.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,4013 @@
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 1999-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 1996-2021 Free Software Foundation, Inc.
+Copyright (C) 2009-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 2006-2021 Free Software Foundation, Inc.
+Copyright (C) 2004-2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/compile
+SPDXID: SPDXRef-item547302
+FileChecksum: SHA1: c37f7649d459d82714935cdd98c686763351f12e
+FileChecksum: SHA256: c207b390aac6323062b982214a6c63448e53e6911107993abe96f35fe7a30a18
+FileChecksum: MD5: b816f92498a6ec8138d9ec53a14bcaa4
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1999-2021 Free Software Foundation, Inc. Written by Tom Tromey <tromey@cygnus.com>. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/configure
+SPDXID: SPDXRef-item547084
+FileChecksum: SHA1: ed9b653d59eeae0a6d53478f29566f72922e422b
+FileChecksum: SHA256: 4eadb229fe88f96bcb5241025681a8ec27a438311d5b216f378eb071167bc2a7
+FileChecksum: MD5: 05d3ddaaffccf560e3bd69d6dd3e8c1a
+LicenseConcluded: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-fsf-free
+FileCopyrightText: <text> Copyright (C) 1992-1996, 1998-2017, 2020-2021 Free Software Foundation, Inc.
+Copyright (C) 2021 Free Software Foundation, Inc.
+Copyright (C) 2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/test-driver
+SPDXID: SPDXRef-item547293
+FileChecksum: SHA1: b995d8e65c0364e5e5ee6860c3f8b31e94064e6a
+FileChecksum: SHA256: 08ff8986c3a0ce8bf32d25ec7e4752ffbce9c89589f018836c97a08b6573ac2c
+FileChecksum: MD5: 761b9d083bd4741484292ce03510f40a
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 2011-2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/queue.h
+SPDXID: SPDXRef-item547321
+FileChecksum: SHA1: d0608b88907c8bd5dcfbc10f18e4a0b02c7cf560
+FileChecksum: SHA256: d55f965dc6a0137ac5d3fe31f9bcc617fe562137d420b21c8de6a4084b556c5d
+FileChecksum: MD5: 92ef4e9cd9086e25ce29ddb8991f6ff3
+LicenseConcluded: LicenseRef-BSD-3-Clause
```

Comment:
Please take care of my comment in the disclosure file about the text of the BSD-3-Clause license